.coveragerc

```diff
@@ -7,10 +7,10 @@ source =
 omit =
     h/debug.py
     h/migrations/*
-    h/cli/commands/create_annotations.py
+    h/cli/*
```

Comment:
This looks good to me but what's the rationale?

Reply:
I don't think we're ever going to write tests for these things. We're more likely to rewrite them completely or remove them.